### ISSUE TITLE
Dalle3 integration [ACD-523]

### ISSUE THREAD

**Issue by Rafal80** [10clouds/aiconsole#291]
Migrated from issue: [ACD-523](https://10clouds.atlassian.net/browse/ACD-523) (Story)

[ACD-523]: https://10clouds.atlassian.net/browse/ACD-523?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by cielecki** [comment]
Done in: https://github.com/10clouds/aiconsole/commit/88bd8642caaa679650267c87eecffd21029ab5c6

**Comment by cielecki** [comment]
@udislau can you do a code review here?

**Comment by cielecki** [comment]
@udislau Added that as a separate task

**Comment by kgradzka** [comment]
Will be retested after this fix: https://github.com/10clouds/aiconsole/issues/376.

**Comment by kgradzka** [comment]
Retesting blocked by: https://github.com/10clouds/aiconsole/issues/636

**Comment by kgradzka** [comment]
Retesting blocked by: https://github.com/10clouds/aiconsole/issues/636
